tests/amsterdam/eip7928_block_level_access_lists/test_block_access_lists_opcodes.py

```diff
@@ -47,6 +47,46 @@
 pytestmark = pytest.mark.valid_from("Amsterdam")
 
 
+@pytest.fixture
+def call_memory_expansion_cost(fork: Fork) -> Callable[[int], int]:
```

Comment:
There's a function in `Fork` that does this and is updated with each fork:
https://github.com/ethereum/execution-specs/blob/cc82819f277c014ef900250f9954c964011389e8/packages/testing/src/execution_testing/forks/base_fork.py#L358-L367